src/openforms/forms/tests/test_import_export.py

```diff
@@ -1502,7 +1577,9 @@ def test_import_form_with_objecttype_uuid_objects_api_registration_backend(self)
                             "name": "Test backend",
                             "backend": "objects_api",
                             "options": {
-                                "objects_api_group": self.objects_api_group.pk,
+                                "objects_api_group": ObjectsAPIGroupConfigFactory.create(
+                                    for_test_docker_compose=True
+                                ).pk,
```

Comment:
I thought we had an issue somewhere for the primary keys in export artifacts, but I can't seem to find it?

Reply:
iirc it was added as a checkbox in this issue: https://github.com/open-formulieren/open-forms/issues/4330